helix-query.yaml

```diff
@@ -6,7 +6,7 @@ version: 1
 indices:
   blog-posts:
     source: html
-    fetch: https://{repo}-{owner}.project-helix.page{path}
+    fetch: https://{ref}--{repo}--{owner}.project-helix.page{path}
```

Comment:
Need to use `ref` as well, otherwise we're requesting our own files from `master`